pkgs/by-name/ke/kew/package.nix

```
@@ -1,28 +1,32 @@
 { lib
 , stdenv
 , fetchFromGitHub
+, pkg-config
 , ffmpeg
 , fftwFloat
 , chafa
 , freeimage
 , glib
-, pkg-config
+, opusfile
+, libopus
+, libvorbis
 }:
 
 stdenv.mkDerivation rec {
   pname = "kew";
-  version = "1.5.2";
+  version = "1.8.1";
 
   src = fetchFromGitHub {
     owner = "ravachol";
     repo = "kew";
     rev = "v${version}";
-    hash = "sha256-Om7v8eTlYxXQYf1MG+L0I5ICQ2LS7onouhPGosuK8NM=";
+    hash = "sha256-WdWeC9Bx7gI55pDtpORVAeld8QWo6BMzxQ/bliSAFII=";
   };
 
   nativeBuildInputs = [ pkg-config ];
-  buildInputs = [ ffmpeg freeimage fftwFloat chafa glib ];
+  buildInputs = [ ffmpeg freeimage libvorbis opusfile.dev libopus.dev fftwFloat chafa glib ];
```

That's not necessary, it does not reduce the final derivation size. Please stick to the common Nixpkgs conventions: suboutputs are rarely used.

```suggestion
  buildInputs = [ ffmpeg freeimage libvorbis opusfile libopus fftwFloat chafa glib ];
```